resources/schema/Operator.json

```diff
@@ -304,6 +304,9 @@
       "properties": {
         "operator": {
           "const": "invalid_duration"
+        },
+        "negative": {
```

Comment:
I think `type` specifications for operator parameters are usually defined in the `properties` section, not within the `anyOf` section that specifies the allowable operations.  Within the `properties` section, the definitions are usually ordered alphabetically, so I'd recommend moving the `type` specification for `negative` to be between the specifications for `name` and `operator` (i.e., somewhere near line 906).

Reply:
Thank you for pointing this out @ASL-rmarshall  I made the changes to the schema and pushed them to remote branch

Reply:
The change looks good.